### PR TITLE
Update Deployment

### PR DESCRIPTION
**Pull request by dijonron**
with latest changes

### CODE REVIEW

File: socialdistribution/api/models.py [hunk 1]
```diff
@@ -161,7 +162,7 @@ def set_post_content(self, title, categories, text_content=None, image_content=N
     # TODO: Description is not very descriptive
     def set_post_metadata(self, author, visibility, unlisted):
         assert self.title, "set_post_content must be called before set_post_metadata"
-
+        logging.error(unlisted)
```
Comment (Chase-Warwick) [inline socialdistribution/api/models.py:165]: I think this is for debugging.

File: socialdistribution/socialdistribution/settings.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 load_dotenv()
 # env variables
 SECRET_KEY = os.environ.get("SECRET_KEY")
-DEBUG = os.environ.get("DEBUG")
+DEBUG = (os.environ.get("DEBUG") == 'True')
```
Comment (Chase-Warwick) [inline socialdistribution/socialdistribution/settings.py:26]: Do we want DEBUG to be true on deployment? Maybe but I thought I would ask

**Review by Chase-Warwick — CHANGES_REQUESTED** [review]
Just a minor debug message still in there

File: socialdistribution/socialdistribution/settings.py (hunk 2 above)
Comment (dijonron) [reply socialdistribution/socialdistribution/settings.py:26]: the Heroku env variables have to be stored as strings, so I think this should check if the 'False' == 'True' on Heroku, so I think it should not be in debug mode when deployed

**Pull request merged**